docs/development_suite/api-console/api-design/services.md

```diff
@@ -451,42 +451,30 @@ Finally, you should create a new environment variable passing the following data
 
 Once saved all these changes, you should see the volume correctly mounted in the generated deployment file.
 
-### Advanced Configuration
+## Raw Manifest configuration
```

Comment:
For coherence, this should still be Advanced configuration (each title refers to the corresponding tab), and the subsections should be about raw manifest mode and service deletion

Reply:
nope, contents are not coherent with tabs